src/js/server/hcapi/stats.js

```diff
@@ -0,0 +1,29 @@
+const { Router } = require('express');
+const { Hacker, ApplicationReview, HackerApplication } = require('js/server/models');
+const { createHttpError } = require('./errors');
+
+const statsRouter = new Router();
+
+statsRouter.get('/', (req, res, next) => {
+
+  // Get:
+  // - The total number of applications
+  // - The total number of sign ups
+  // - The number of reviews (note that each application needs to be reviewed twice)
```

Comment:
Are there plans to account for this in the stats? I feel like "400 out of 700 applications reviewed" is a little more helpful than "800 reviews made for 700 applications".